src/github/client/github-client.ts

```diff
@@ -26,5 +30,42 @@ export class GitHubClient {
 			this.restApiUrl = `${baseUrl}/api/v3`;
 			this.graphqlUrl = `${baseUrl}/api/graphql`;
 		}
+
+		this.axios = axios.create({
+			baseURL: this.restApiUrl,
+			transitional: {
+				clarifyTimeoutError: true
+			},
+			... this.getProxyConfig(this.restApiUrl)
+		});
 	}
+
+	public getProxyConfig = (baseUrl: string): Partial<AxiosRequestConfig> => {
+		if (baseUrl
+			.replace(RegExp("/$"), "")
+			.endsWith("atlassian.com")) {
+			return this.noProxyConfig();
+		} else {
```

Comment:
don't need the else if you're returning in the if

Reply:
✅